src/themes/shared/typography.ts

```diff
@@ -101,4 +101,10 @@ export const typography = {
     lineHeight: lineHeights.sm,
     fontSize: fontSizes.sm,
   },
+  label: {
+    fontFamily: fonts.makeItSans,
+    fontWeight: fontWeights.bold,
+    lineHeight: lineHeights.sm,
+    fontSize: fontSizes.base,
```

Comment:
I'm wondering if it's possible to use the `header4` here instead adding this new token. Maybe the name doesn't match at all but rather we will repeat so many tokens.  What do you think?

Reply:
I didn't because I was afraid of coupling this, but maybe it was premature optimisation. Happy to change that if you think it makes sense.

Reply:
We can leave it as you did. I think, the label fits better as the `heading4`.
But we can watch out the typography increase. Otherwise, we will need to doble check the typography token approach. Thanks!

Reply:
We will update figma and figma-tokens with this new token 🙌🏼